tasks/move-granules/tests/index-test.js

```diff
@@ -160,33 +160,71 @@ test.serial('should overwrite files', async (t) => {
     name: filename
   }];
 
+  const uploadParams = {
+    Bucket: t.context.stagingBucket,
+    Key: sourceKey,
+    Body: 'Something'
+  };
+
+  console.log(`CUMULUS-970 debugging: start s3 upload. params: ${JSON.stringify(uploadParams)}`);
+
   await promiseS3Upload({
     Bucket: t.context.stagingBucket,
     Key: sourceKey,
     Body: 'Something'
   });
 
+  console.log(`CUMULUS-970 debugging: start move granules. params: ${JSON.stringify(newPayload)}`);
+
   let output = await moveGranules(newPayload);
+
+  console.log('CUMULUS-970 debugging: move granules complete');
+
   await validateOutput(t, output);
+
+  console.log(`CUMULUS-970 debugging: head object destKey ${destKey}`);
+
   const existingFile = await headObject(
     t.context.publicBucket,
     destKey
   );
 
+  console.log('CUMULUS-970 debugging: headobject complete');
+
   // re-stage source jpg file with different content
   const content = randomString();
+
+  const uploadParams2 = {
```

Comment:
We could just do `uploadParams.Body = content` here and pass `uploadParams` to `promiseS3Upload`

Reply:
done